sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -203,6 +203,12 @@
  * {@link ProducerConfig} for sink. E.g. if you would like to enable offset
  * <em>auto commit</em> (for external monitoring or other purposes), you can set
  * <tt>"group.id"</tt>, <tt>"enable.auto.commit"</tt>, etc.
+ *
+ * <h3>Event Timestamp and Watermark</h3>
+ * The default event_timestamp and watermark increases along as processing timestamp,
```

Comment:
minor: s/Event/Record/ ?
"By default record timestamp and watermark are based on processing time in KafkaIO reader. This can be overridden by providing  {@code WatermarkFn} with {@link Read#withWatermarkFn(SerializableFunction)}, and {@code TimestampFn} with {@link Read#withTimestampFn(SerializableFunction)}. Note that {@link KafkaRecord#getTimestamp()} reflects timestamp provided by Kafka if any, otherwise it is set to processing time."